docs/pages/component_overview.js

```diff
@@ -0,0 +1,163 @@
+    case 'Actions':
+    case 'Avatars':
+    case 'Controls':
+    case 'Data':
+    case 'Fields & Forms':
+    case 'Loading':
+    case 'Messaging':
+    case 'Navigation':
+    case 'Pins & Imaginery':
+    case 'Structure':
+    case 'Text':
+      return 'green-matchacado-0';
+    default:
+      return 'green-matchacado-0';
+  }
+};
+
+// GENERAL_COMPONENT_LIST is an array with component data. Each array item contains the SVG data and other metadata such as the component category. The following reduce method processes the GENERAL_COMPONENT_LIST array into an object grouping and mapping components per category so that we can map per category and pass each category value to <List />.
+const GENERAL_COMPONENT_CATEGORY_MAP = GENERAL_COMPONENT_LIST.reduce(
+  (accumulatedMap, currentItem) => {
+    const copyAccumulatedMap = { ...accumulatedMap }; // This copy prevents Eslint from complaining about reassigning
+
+    if (copyAccumulatedMap[currentItem.category]) {
+      copyAccumulatedMap[currentItem.category] = [
+        ...accumulatedMap[currentItem.category],
+        currentItem,
+      ];
+    } else {
+      copyAccumulatedMap[currentItem.category] = [currentItem];
+    }
+    return copyAccumulatedMap;
+  },
+  {},
+);
+
+export type ListItemType = Array<{|
+  svg: Element<typeof Accessibility>,
+  name: string,
+  description: string,
+  category: string,
+  path?: string,
+  hasDarkBackground?: boolean,
+|}>;
+
+function List({ array, title = '' }: {| array: ListItemType, title?: string |}): Node {
+  return (
+    <IllustrationSection title={title} grid="auto-fill" min={312}>
+      {array
+        .sort((a, b) => {
+          if (a.name < b.name) {
+            return -1;
+          }
+          if (a.name > b.name) {
+            return 1;
+          }
+          return 0;
+        })
+        .map((element, idx) => (
+          <IllustrationCard
+            key={idx}
+            href={element?.path ?? `/${element.name.replace(/\s/g, '_').toLowerCase()}`}
+            title={element.name}
+            description={element.description}
+            color={getIllustrationCardColor(element.category, element?.hasDarkBackground)}
+            image={element.svg}
+          />
+        ))}
+    </IllustrationSection>
+  );
+}
+
+export default function ComponentOverview(): Node {
+  const [order, setOrder] = useState('category');
+
+  return (
+    <Page title="Component overview" hideSideNav hideEditLink>
+      <Flex direction="column" width="100%">
+        <IllustrationContainer justifyContent="start">
+          <PageHeader
+            name="Component overview"
+            description="Not sure which component you need? Take a look below or set up time with the Gestalt team."
+            showSourceLink={false}
+          />
+        </IllustrationContainer>
+        <IllustrationContainer justifyContent="start">
+          <Flex gap={2}>
+            <Box aria-hidden>
+              <Text>Sort by</Text>
```

Comment:
This doesn't look vertically in line with the option text:
![Screen Shot 2022-03-24 at 1 46 43 PM](https://user-images.githubusercontent.com/12059539/160007508-62ac2f51-544b-4535-87f5-8e0c6a917d79.png)

Reply:
DONE